modules/create_timepoint/php/create_timepoint.class.inc

```diff
@@ -243,6 +262,16 @@ class Create_Timepoint extends \NDB_Form
             $errors['psc'] = "Site must be selected from the available dropdown.";
         }
 
+        //Project
+        $projects = \Utility::getProjectList();
+        if (count($projects) == 0) {
+            $errors['project'] = "No projects are defined in the database, please add
+            projects in order to continue";
+        } else if (count($projects) > 1 && is_null($val['project'])) {
+            $errors['project'] = "A project must be selected from the available 
+            dropdown.";
```

Comment:
Can you normalize the whitespace (newline and a bunch of spaces) in the middle of these strings? Either move them onto 1 line or use `.` to concatenate 2 strings if it's because of phpcs line length requirements.

Reply:
@driusan done